example/src/App.tsx

```diff
@@ -204,7 +205,11 @@ export default function App() {
         <RootStack.Screen
           name="BasicPictureInPicture"
           component={BasicPictureInPicture}
-          options={{ title: 'Basic Picture in Picture' }}
+          options={{
+            title: 'Basic Picture in Picture',
+            // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
This looked fishy to me, but after looking at the [component documentation](https://reactnavigation.org/docs/header-buttons/) I guess is this is the way to do it.

Reply:
I know and I couldn't find any other way where the warning doesn't show up.